src/main.rb

```diff
@@ -0,0 +1,19 @@
+def main
+  puts 'Welcome to My Catalog of Things'
+
+  loop do
+    puts "\nPlease choose an option by entering a number
+    1. Get genre
+    2. Get author
+    3. Get source
+    4. Get label
+    5. Get published date
+    6. Exit"
+    input = gets.chomp.to_i
```

Comment:
- [ ] Kindly complete the options that are shown in the project requirements
![image](https://user-images.githubusercontent.com/103154117/218778040-a9450fa5-b7d5-44a3-934a-e82c9abb267f.png)